cmd/fyne/internal/commands/init.go

```diff
@@ -106,11 +119,20 @@ func checkFileOrCreate(file, content string) error {
 
 func initAction(ctx *cli.Context) error {
 	modpath := ctx.Args().Get(0)
-	appID := ctx.Args().Get(1)
-	appName := ctx.Args().Get(2)
+	appID := ctx.String("appID")
+	appName := ctx.String("name")
 
 	if modpath == "" {
-		return cli.ShowSubcommandHelp(ctx)
+		modpath = "example"
+
+		wd, err := os.Getwd()
+		if err != nil {
+			return err
+		}
+
+		if wd != "" && wd != "." {
```

Comment:
In the case of "." (is that really returned buy the `Getwd`?) can we try to get the absolute path to then take the `Base` of?

Reply:
After checking the `os.Getwd()` documentation and code, I don't think it returns a dot. Removing the check. Otherwise it should do what you describe.